pkg/aws/manager_rds_subnet_group.go

```diff
@@ -0,0 +1,105 @@
+package aws
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/rds"
+	"github.com/integr8ly/cluster-service/pkg/clusterservice"
+	"github.com/integr8ly/cluster-service/pkg/errors"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	loggingKeySubnetGroup = "subnet-group-name"
+)
+
+var _ ClusterResourceManager = &RDSSubnetGroupManager{}
+
+type RDSSubnetGroupManager struct {
+	rdsClient rdsClient
+	logger    *logrus.Entry
+}
+
+func NewDefaultRDSSubnetGroupManager(session *session.Session, logger *logrus.Entry) *RDSSubnetGroupManager {
+	fmt.Println("creating new RDS Subnet Group Manager")
```

Comment:
I believe this line is only for debugging purposes?

Reply:
Good catch